Project.toml

```diff
@@ -80,6 +81,7 @@ StaticArrays = "1"
 StrideArrays = "0.1.18"
 StructArrays = "0.6"
 SummationByPartsOperators = "0.5.25"
+T8code = "0.3.0"
```

Comment:
There is also T8code.jl v0.4. Is this version compatible with it?

Reply:
Updated to "0.4.0".